components/Comment/Comment.tsx

```diff
@@ -54,13 +61,21 @@ const Comment = ({
             <CommentHeader
               authorProfile={comment.createdBy.authorProfile}
               comment={comment}
+              handleEdit={() => setIsEditMode(!isEditMode)}
             />
           </div>
           {isEditMode ? (
             <CommentEditor
-              handleSubmit={({ content }) => handleUpdate({ comment, content })}
+              handleSubmit={async (args) => {
+                console.log("args", args);
+                await handleUpdate(args);
+                setIsEditMode(false);
+              }}
               content={comment.content}
+              commentId={comment.id}
+              author={currentUser?.authorProfile}
               editorId={`edit-${comment.id}`}
+              handleClose={() => _handleCloseEdit()}
```

Comment:
in these cases, if we aren't passing any params, can we pass it like this instead:

```
handleClose={_handleCloseEdit}
```